tumor-only-benchmarking/Dockerfile

```diff
@@ -1,13 +1,17 @@
 FROM ubuntu:22.04
 LABEL maintainer="Saksham Phul (phuls@chop.edu)"
-LABEL description="RTG tool for benchmarking vcf files"
+LABEL description="RTG tool for benchmarking vcf files & bcftools to play with VCFs, pandarallel to speed up pandas dataframes"
 
 ENV DEBIAN_FRONTEND noninteractive
-
+ENV BCFTOOLS_VERSION 1.16
```

Comment:
https://github.com/samtools/bcftools has a 1.17 release. Probably want to use that.